.travis.yml

```diff
@@ -36,9 +36,6 @@ install: git clone --depth 1 https://github.com/bats-core/bats-core.git bats
 # Default for auto-generated jobs.
 script: make test-build
 
-after_script: |
```

Comment:
Without this and bats now every CI run has a lot of output printed instead of being in a collapsible section in the Travis CI interface - compare [this](https://travis-ci.org/pyenv/pyenv/jobs/520866465) (post-change) and [this](https://travis-ci.org/pyenv/pyenv/jobs/517222222) (pre-change). In the latter case some 9000 lines are hidden by default, which is nice.

Reply:
Ah, to be honest, I didn't think much about Travis CI's interface 😞  At least there would be something we can/should improve.

In fact, at that time what I wanted to do was to avoid travis's build timeout after non-output for 10 minutes. There could be even more better workaround for the purpose at least.

https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received